distribution/docs/src/main/resources/content/_configuring/uploading-and-editing.adoc

```diff
@@ -0,0 +1,35 @@
+:title: Configuring User Access to Ingest and Metadata for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 02
+:summary: Configuring user ability to upload products and edit existing metadata in ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} lets the administrator control user access to ingest and metadata. The administrator
+can show or hide the uploader, letting them control whether the user can ingest products. They
```

Comment:
:pencil2: `The administrator choose to show the uploader, which allows users to ingest products through ${catalog-ui}.`